README.md

```diff
@@ -15,7 +15,7 @@ After these are installed, you may install `curvelops` with:
 ```bash
 export FFTW=/path/to/fftw
 export FDCT=/path/to/CurveLab
-python3 -m pip install git+https://github.com/PyLops/curvelops@main
+python3 -m pip install git+https://github.com/PyLops/curvelops@0.21
```

Comment:
What about changing this with the recipe you provide in the PR comment?

Reply:
I needed to release the tag first!